### ISSUE TITLE
Blank values for user fields shouldn't be accepted

### ISSUE THREAD

**Issue by johnbillion** [wp-cli/entity-command#36]
Given the following command, a success message is shown, when in fact nothing was updated.

```shell
wp user update admin --user_email
```

Is there a purpose to allowing empty `--user_*` fields in this command?

**Comment by danielbachhuber** [comment]
All arguments are passed through to `wp_update_user()`. Given `wp_update_user()` doesn't return any error when non-sensical arguments are passed, `wp user update` doesn't produce an error.

**Comment by miya0001** [comment]
> Given wp_update_user() doesn't return any error when non-sensical arguments are passed, wp user update doesn't produce an error.

Ah, I tried to fix this problem.
Should we fix this problem on WP-CLI? I want to know how do we handle with similar case.
I understood that validation should not be added to wp-cli because WordPress core doesn't have it, right?

**Comment by danielbachhuber** [comment]
This is fixed by #39 

```
$ wp user update 1 --user_email
Warning: --user_email parameter needs a value
Error: Need some fields to update.
```
